src/Microsoft.Management.UI.Internal/HelpWindow/HelpViewModel.cs

```diff
@@ -88,9 +88,9 @@ public double Zoom
             set
             {
                 this.zoom = value;
-                this.OnNotifyPropertyChanged("Zoom");
-                this.OnNotifyPropertyChanged("ZoomLabel");
-                this.OnNotifyPropertyChanged("ZoomLevel");
+                this.OnNotifyPropertyChanged(nameof(Zoom));
```

Comment:
We should exclude Microsoft.Management.UI.Internal. Can we add global config in the folder with defaults?

Reply:
I've added a project AnalyzerConfig with the following: `dotnet_analyzer_diagnostic.severity = none`.